package.json

```diff
@@ -36,7 +36,8 @@
     "concat-stream": "^2.0.0",
     "json-stable-stringify": "^1.0.1",
     "request": "^2.88.2",
-    "rpc-websockets": "^7.4.8"
+    "rpc-websockets": "^7.4.8",
+    "ws": "7.4.3"
```

Comment:
I am not sure if we need to import `ws` just for the type. It would just make `holo-envoy` heavy.

Reply:
I made sure to use the same version that `rpc-websockets` uses, so the it's not increasing the dependency tree at the moment. Would you rather than I depend on `@types/ws`?

Reply:
Both are in the dependency tree so either work. :+1: